README.md

```diff
@@ -34,42 +44,36 @@ Ok, two modules loaded.
 *Main>
 ```
 
-Good Examples:
-Assume you have followed the instructions above which run GHCi and load the Main module
+Then, execute function `expr` with two arguments:
 
-```
-*Main> goodAddTwoInteger
-Interpreting Program:
-(+ 1 2)
--> 3
-```
+- First argument: the AST of the targeted language. Please refer to `Grammar.hs` for details about what a correct AST of **Lil' Lisp** looks like)
 
+- Second argument: `[]` (this is the initial stack when the program first starts with)
+
+An example, let say we'd like to test the interperter on the following Lil' Lisp program
```

Comment:
> An example, **let say** we'd like to test the **interperter** on the following Lil' Lisp program

~~Vibe~~ Spell check!

Change `An example` too `For example`, `let say` to `let's say`, `interperter` to `interpreter`